db/migrate/20220811091450_create_likes.rb

```diff
@@ -0,0 +1,9 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
+      t.references :post, null: false, foreign_key: true
```

Comment:
- [ ] Same here, user should be called as author based on EDR diagram: 

`t.references :author, foreign_key: { to_table: 'users' }`
this will create author_id and a corresponding index

- [ ] OPTIONAL, you don't have to explicitly write `null: false` when using foreign_key: true, because foreign_key guarantees referencial integrity which means that it will never be null

Reply:
Implemented 👍